content/article/obscure-perl-trick--single-quote-separators.md

```diff
@@ -17,7 +17,7 @@ One of the delights of working in an old language with penchant for backwards-co
    > the old-fashioned syntax is still supported for backwards
    > compatibility
 
-How interesting! Ada uses a single quote as an [attribute delimiter](https://en.wikibooks.org/wiki/Ada_Programming/Delimiters/'), similar to the possessive in English:
+How interesting! Ada uses a single quote as an [attribute delimiter](https://en.wikibooks.org/wiki/Ada_Programming/Delimiters/\)'), similar to the possessive in English:
```

Comment:
This might look strange but the correct link seems to have a ')' at the end

https://en.wikibooks.org/wiki/Ada_Programming/Delimiters/)

Reply:
The link is to the "single quote" delimiter and the original link should probably be URL-encoded like this: https://en.wikibooks.org/wiki/Ada_Programming/Delimiters/%27

The link the the ) delimiter is something else.

Reply:
If you fix up the pull request I can merge it and you can get Hacktoberfest credit. Or I can just make the changes on my own.